src/pages/Admin/Venue/AnnouncementOptions/AnnouncementOptions.scss

```diff
@@ -0,0 +1,20 @@
+@import "scss/constants";
+
+.AnnouncementOptions {
+  display: flex;
+  justify-content: space-between;
+  padding-top: $spacing--lg;
+
+  &__left-side {
+    font-size: $font-size--lg;
+    margin-top: $spacing--xs;
+  }
+}
+
+// Override global styles button
+button.AnnouncementOptions__edit-button {
+  height: $submit-button-diameter;
+  width: $button-width;
+  background-color: $white;
+  color: $primary--live;
+}
```

Comment:
As called out elsewhere, better patterns for overriding this? @sunny-viktoryia @mike-lvov @denisdimitrov

Reply:
gave the reference to my workaround here: https://github.com/sparkletown/sparkle/pull/1440#discussion_r658658304

Reply:
fixed 1c2d835